Library/Homebrew/rubocops/urls.rb

```diff
@@ -66,6 +66,11 @@ def audit_formula(_node, _class_node, _parent_class_node, body_node)
             problem "#{url} should be `https://www.apache.org/dyn/closer.lua?path=#{match[1]}`"
           end
 
+          version_control_pattern = %r{^(cvs|bzr|hg|fossil)://}
```

Comment:
Not sure about where to best put it in this file. Any input appreciated.

Reply:
This seems fine to me! Alternative would be a `VERSION_CONTROL_PATTERN = %r{^(cvs|bzr|hg|fossil)://}.freeze` at the class level.

Reply:
I see, thanks for pointing out.

Btw, I also tried this, to have all the vcs stuff combined:

```
version_control_patterns = Regexp.union([%r{^(cvs|bzr|hg|fossil)://}, 
                                         %r{^(svn)\+http://}])
audit_urls(urls, version_control_patterns) do |match, _|
  problem "Use of the #{match[0]} scheme is deprecated, pass `:using => :#{match[1]}` instead"
end
```

But then the problem is that the matching `svn` is not contained in `match[1]`, but rather `match[2]`. I'm not an expert with `Regexp.union`. Maybe you have an idea to optimize this code :)

Reply:
Yeh, I've tried and can't find a way to match them consistently unfortunately. Two regex seems fine 👍

Reply:
I used https://rubular.com for testing in case that's useful to you.